run/log.go

```diff
@@ -65,13 +58,13 @@ var (
 
 // Initialize package-global logger variables.
 func init() {
-	wallet.UseLogger(walletLog)
+	/*wallet.UseLogger(walletLog)
```

Comment:
what's wrong with this? Maybe we can pass logger when creating a wellet instance?